libs/async/include/hpx/async/applier/apply_helper.hpp

```diff
@@ -77,9 +77,8 @@ namespace hpx { namespace applier { namespace detail {
 #endif
 #endif
         data.priority = fix_priority<Action>(priority);
-        data.stacksize =
-            threads::get_stack_size(static_cast<threads::thread_stacksize>(
-                traits::action_stacksize<Action>::value));
+        data.stacksize = static_cast<threads::thread_stacksize>(
+            traits::action_stacksize<Action>::value);
```

Comment:
Should we make `traits::action_stacksize<Action>::value` to be of type `enum thread_stacksize` instead of the `static_cast` (I know this is old code, but I thought I'd mention it anyways)?

Reply:
Good point. It seems the action stacksize is already using the thread stacksize enum, but inside another enum... Will change this.